LICENSE.txt

```diff
@@ -1246,6 +1550,110 @@ github.com/klauspost/compress is under the Apache-2.0 license.
 
 --------------------------------------------------------------------------------
 
+3rdparty dependency github.com/klauspost/compress
+is statically linked in certain binary distributions, like the Python wheels.
+github.com/klauspost/compress is under the BSD-3-Clause license.
```

Comment:
They may be duplicated entries.

Reply:
Ah, I think go-licenses detected multiple licenses for some projects and duplicated the entries for each license